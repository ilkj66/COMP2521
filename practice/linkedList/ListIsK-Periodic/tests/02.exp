Enter list: Enter k: 
List: [2] -> [2] -> [2] -> [2] -> [2] -> [2] -> [3] -> X
listIsKPeriodic returned FALSE
