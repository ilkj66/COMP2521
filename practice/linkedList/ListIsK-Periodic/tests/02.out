Enter list: Enter k: 
list: [2] -> [2] -> [2] -> [2] -> [2] -> [2] -> [3] -> X
listIsKPeriodic returned FALSE
